code/modules/mob/living/simple_animal/friendly/mule_bot/mule_bot.dm

```diff
@@ -0,0 +1,202 @@
+
+/obj/effect/mule_link/Initialize()
+	. = ..()
+	var/mob/living/simple_animal/mule_bot/M = new(loc)
+	var/obj/item/remote/R = new(loc)
+	M.try_link(R)
+	R.bot = M
+	qdel(src)
+
+
+/mob/living/simple_animal/mule_bot
```

Comment:
Write a comment explaining the decision to use simple animals pls

Reply:
Like I said in discord other then "needing" ai behavior there is not insane amount of reasons why. I just need something that I can controll preferbly with the movement system to avoid complicated move stuff

Reply:
Yea just write a comment so anyone looking after understands the decision behind doing this